packages/opentelemetry-instrumentation-openai/tests/traces/test_assistant.py

```diff
@@ -47,26 +49,26 @@ def test_new_assistant(exporter, openai_client, assistant):
         "openai.assistant.run",
     ]
     open_ai_span = spans[0]
-    assert open_ai_span.attributes["llm.request.type"] == "chat"
-    assert open_ai_span.attributes["gen_ai.request.model"] == "gpt-4-turbo-preview"
-    assert open_ai_span.attributes["gen_ai.response.model"] == "gpt-4-turbo-preview"
+    assert open_ai_span.attributes[SpanAttributes.LLM_REQUEST_TYPE] == "chat"
+    assert open_ai_span.attributes[SpanAttributes.LLM_REQUEST_MODEL] == "gpt-4-turbo-preview"
+    assert open_ai_span.attributes[SpanAttributes.LLM_RESPONSE_MODEL] == "gpt-4-turbo-preview"
     assert (
-        open_ai_span.attributes["gen_ai.prompt.0.content"]
+        open_ai_span.attributes[f"{SpanAttributes.LLM_PROMPTS}.0.content"]
```

Comment:
This is ok, just because this will be refactored into events pretty soon so no need to change it and the likes of it